src/Forms/FormInput.js

```diff
@@ -24,11 +31,18 @@ FormInput.displayName = 'FormInput';
 FormInput.propTypes = {
     className: PropTypes.string,
     disabled: PropTypes.bool,
+    id: PropTypes.string,
+    name: PropTypes.string,
+    placeholder: PropTypes.string,
     readOnly: PropTypes.bool,
-    state: PropTypes.oneOf(INPUT_TYPES)
+    state: PropTypes.oneOf(INPUT_TYPES),
+    type: PropTypes.string,
+    value: PropTypes.string
 };
 
 FormInput.propDescriptions = {
+    disabled: 'Set to **true** to mark component as disabled and make it non-interactive.',
+    readOnly: 'Set to **true** to mark component as readonly.',
```

Comment:
those are default props - their descriptions will be assigned automatically by https://github.com/SAP/fundamental-react/blob/master/src/_playground/documentation/Properties/defaults.js

You will need add descriptions here for `name`, `state`, and arguably `type` as the definition differs here.

Reply:
Ok, I want to make sure I understand you correctly.

The descriptions at [https://github.com/SAP/fundamental-react/blob/master/src/_playground/documentation/Properties/defaults.js](https://github.com/SAP/fundamental-react/blob/master/src/_playground/documentation/Properties/defaults.js) are automatically added.

What I need to provide are the descriptions for the props (`name`, `value`) that are not described in the above file as well as modify `type`'s description to reflect the FormInput component.